mysql-test/suite/gcol/t/innodb_virtual_debug_purge.test

```diff
@@ -104,6 +104,7 @@ INSERT INTO t1(a, b) VALUES (5, 5);
 SET DEBUG_SYNC= 'inplace_after_index_build SIGNAL uncommitted WAIT_FOR purged';
 --error ER_ALTER_OPERATION_NOT_SUPPORTED_REASON
 ALTER TABLE t1 ADD COLUMN c INT GENERATED ALWAYS AS(a+b), ADD INDEX idx (c), ALGORITHM=INPLACE, LOCK=NONE;
+sleep 2;
 send ALTER TABLE t1 ADD COLUMN c INT GENERATED ALWAYS AS(a+b), ADD INDEX idx (c), ALGORITHM=INPLACE, LOCK=SHARED;
```

Comment:
The alter statement will create an readview, and the value of `ReadViewBase::m_low_limit_no` is obtained from `SCN_Mgr::m_safe_limit_no`, which is refreshed every second. Here `sleep` is used to wait `SCN_Mgr::m_safe_limit_no` to be refreshed, otherwise the subsequent `wait_all_purged.inc` will be blocked by this alter statement.